docs/source/cc_service.md

```diff
@@ -0,0 +1,297 @@
+# Chaincode as an external service
+
+Fabric v2.0 supports chaincode deployment and execution outside of Fabric that enables users to manage a chaincode runtime independently of the peer. This facilitates deployment of chaincode on Fabric cloud deployments such as Kubernetes. Instead of installing the chaincode on every peer, chaincode can now run as a service as opposed to a Fabric launched client. This capability leverages the Fabric v2.0 external builder and launcher functionality which enables operators to extend a peer with programs to build, launch, and discover chaincode. Before reading this topic you should become familiar with the [External Builds and Launchers](./cc_launcher.html) content.
+
+Prior to the availability of the external build and launcher functionality, the chaincode package would contain the source code for the chaincode. The new external build and launcher functionality now allows users to optionally customize the build process. And with respect to running the chaincode as an external service, the build process allows you to specify the endpoint information of the server where the chaincode is running. Hence the package simply consists of the externally running chaincode server endpoint information and TLS artifacts for secure connection. TLS is optional but highly recommended for all environments except a simple test environment.
```

Comment:
> Prior to the availability of the external build and launcher functionality, the chaincode package would contain the source code for the chaincode.

This is a bit disingenuous.  I would say "Prior to the availability of the external builders, the chaincode package content was required to be a set of source code files for a particular language which could be built and launched as a chaincode binary."

With the availability of external launchers, I expect many packages will continue to follow exactly this pattern, it is simply no longer a requirement.

Reply:
Right, didn't mean to imply that original packaging alternative is gone.

Reply:
done